projects/packages/forms/tests/php/contact-form/Feedback_Test.php

```diff
@@ -3140,4 +3140,29 @@ public function test_get_country_flag() {
 
 		remove_filter( 'jetpack_get_country_from_ip', $filter_callback, 10 );
 	}
+
+	/**
+	 * Minimal: submission with first-name/last-name sets author name and first/last getters.
+	 */
+	public function test_author_first_last_on_submission() {
+		// Form can be minimal; we rely on explicit first/last ids in post data.
+		$form = new Contact_Form(
+			array(
+				'title'       => 'Test Form',
+				'description' => 'This is a test form.',
+			),
+			"[contact-field label='Message' type='textarea' required='1'/]"
```

Comment:
I think this should never work. Since the first and and last name fields are not specified in the form.